src/cli/commands/floodsub/pub.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const utils = require('../../utils')
+const debug = require('debug')
+const log = debug('cli:floodsub')
+log.error = debug('cli:floodsub:error')
+
+module.exports = {
+  command: 'pub <topic> <message>',
```

Comment:
I think we should use `<data>` here as we do in go-ipfs (https://github.com/ipfs/go-ipfs/blob/master/core/commands/pubsub.go#L198), just to make it clear that there isn't a *specific* type of message but rather one can send any data as the message. Not super settled on this, so take it as a proposal :)